connector/mqtt/vernemq/v2k-bridge/README.md

```diff
@@ -0,0 +1,66 @@
+# **V2K-bridge**
+The V2K-bridge service implements a bridge between VerneMQ broker and Kafka broker. Basically, it subscribes to some MQTT topics and forwards the messages to some Kafka topics following the dojot's topics rules. In order to scale the bridge service, shared MQTT subscriptions are used, which allows to instantiate a group of consumers, i.e. a group of bridge instances, when necessary. The communication between the bridge and the VerneMQ is secured with mutual TLS. Soon, the communication with Kafka will also use mutual TLS.
+
+<p align="center">
+<img src="./doc/diagrams/bridge.png">
+</p>
+
+# **Configuration**
+
+## **Environment Variables**
+
+Before running the V2K-bridge service within your environment, make sure you configure the environment variables to match your needs.
+
+Key                                | Purpose                                                                          | Default Value                        | Valid Values                                      |
+---------------------------------- | -------------------------------------------------------------------------------- | ------------------------------------ | ------------------------------------------------- |
+BASE_DIR                           | Base directory where the project is located                                      | /opt/v2k_bridge                      | string                                            |
+DATA_BROKER_ADDRESS                | Address of the data broker service                                               | data-broker:80                       | hostname/IP:port                                  |
+EJBCA_ADDRESS                      | Address of the EJBCA service                                                     | localhost:5583                       | hostname/IP:port                                  |
+HOSTNAME                           | Hostname to be used in the certificate common name                               | broker                               | hostname/IP                                       |
+KAFKA_BROKER_LIST                  | Addresses of the kafka brokers separated by a comma                              | kafka-server:9092                    | hostname/IP:port                                  |
+V2K_BACKPRESSURE_MAX_QUEUE_LENGTH  | Maximum backpressure queue length in bytes                                       | 1048576                              | integer                                           |
+V2K_BACKPRESSURE_PARALLEL_HANDLERS | Number of parallel handlers for processing the messages in the backpressure queue| 1                                    | integer                                           |
+V2K_LOG_LEVEL                      | Log level                                                                        | info                                 | silly, debug, verbose, http, info, warning, error |
```

Comment:
https://github.com/dojot/dojot/issues/1475

Reply:
The standard for environment variable names will be discussed later, for now I'm going to stay with this name.